validation/linux_process.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"github.com/opencontainers/runtime-tools/validation/util"
+)
+
+func main() {
+	g := util.GetDefaultGenerator()
+	g.SetProcessNoNewPrivileges(true)
```

Comment:
It looks like this is the only thing specific to this test.  If so, maybe it should be `process_nonewprivileges.go` with a check to `t.Skip` if `runtime.GOOS != "linux"`?  If the goal is to exercise all of the Linux-specific `process` properties in one file, you also need to touch [`apparmorProfile`, etc][1].

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config.md#linux-process

Reply:
I prefer to integrate them into one file. updated.

Reply:
> I prefer to integrate them into one file. updated.

You still aren't testing `apparmorProfile` or `selinuxLabel`.  That's technically ok, because the spec places no runtime conditions on them, but we may want to set them here anyway and leave it to `runtimetest` to `Skip` warnings for any surprising (but still technically conformant) runtime behavior).

Reply:
updated, @wking @liangchenye PTAL.